oauth-client-credentials.html.md.erb

```diff
@@ -0,0 +1,54 @@
+---
+title: OAuth Client Credentials 
+owner: API
+---
+
+<strong><%= modified_date %></strong>
+
+## <a id='overview'></a> OAuth Client Credentials Tutorial
+
+**Note: The Swisscom OAuth is only usable if it is approved and granted from the API team - contact us via the [contact form](https://developer.swisscom.com/support)**
```

Comment:
Please use the `note` class for notes and open external links with the `target="_blank"`:
```
<p class="note">
  <strong>Note</strong>: The Swisscom OAuth is only usable if it is approved and granted by the API team - contact us via the <a href="https://developer.swisscom.com/support" target="_blank">contact form</a>.
</p>
```